gcloud/bigquery/job.py

```diff
@@ -129,6 +129,15 @@ def __init__(self, name, client):
         self.name = name
 
     @property
+    def type(self):
```

Comment:
There isn't but it'd be "better" to use `self.__class__._JOB_TYPE`, no?

Also, can we use a name different than `type`? In general, I try to avoid builtin names.